CPP-007/cpp-007.xml

```diff
@@ -0,0 +1,580 @@
+                </cpp:triggerEvent>
+                <cpp:problemStatement>
+                    <p>Files must be scanned for viruses as part of the 
+                        ingest pipeline to protect the TDA from viruses
+                    </p>
+                </cpp:problemStatement>
+                <cpp:proposedSolution>
+                    <p>Python script to detect viruses using ClamAv virus 
+                        scanner<br /><br />
+                        def check_virus(path):<br />
+                        """Scan files in directory with ClamAV virus scanner.<br />
+                        ...
+                    </p>
+                </cpp:proposedSolution>
+            </cpp:useCase>
+        </cpp:useCases>
+        <cpp:publicDocumentation>
+            <cpp:institution>
+                <cpp:institutionLabel>TIB – Leibniz Information Centre for Science and Technology and University Library</cpp:institutionLabel>
+                <cpp:institutionCountry>DE</cpp:institutionCountry>
+                <cpp:institutionType>National library</cpp:institutionType>
+                <cpp:institutionType>Non-commercial digital preservation service</cpp:institutionType>
+                <cpp:institutionType>Research infrastructure</cpp:institutionType>
+                <cpp:institutionType>Research performing organisation</cpp:institutionType>
+            </cpp:institution>
+            <cpp:documentationLanguage>en</cpp:documentationLanguage>
+            <cpp:linkToDocumentation>
+                <cpp:hyperlink>
+                    https://wiki.tib.eu/confluence/spaces/lza/pages/93608618/Ingest</cpp:hyperlink>
+            </cpp:linkToDocumentation>
+        </cpp:publicDocumentation>
+        <cpp:publicDocumentation>
+            <cpp:institution>
+                <cpp:institutionLabel>CSC – IT Center for Science Ltd.</cpp:institutionLabel>
+                <cpp:institutionCountry>FI</cpp:institutionCountry>
+                <cpp:institutionType>Non-commercial digital preservation service</cpp:institutionType>
+            </cpp:institution>
+            <cpp:documentationLanguage>en</cpp:documentationLanguage>
+            <cpp:linkToDocumentation>
+                <cpp:hyperlink>
+                    https://digitalpreservation.fi/en/services/quality_reports/2024</cpp:hyperlink>
+                <cpp:comment>Monitoring of the Digital Preservation Services: 
+                    "Up-to-date status of the virus check database"
+                </cpp:comment>
+            </cpp:linkToDocumentation>
+        </cpp:publicDocumentation>
+        <cpp:publicDocumentation>
+            <cpp:institution>
+                <cpp:institutionLabel>Archivematica</cpp:institutionLabel>
+                <cpp:institutionCountry>CA</cpp:institutionCountry>
+                <cpp:institutionType>Digital preservation system</cpp:institutionType>
+            </cpp:institution>
+            <cpp:documentationLanguage>en</cpp:documentationLanguage>
```

Comment:
I deleted this element and changed it to an attribute xml:lang attached to the linkToDocumentation element. Could you please apply this change to all these elements?